CHANGELOG.md

```diff
@@ -13,11 +13,15 @@ Documentation for HIPIFY is available at
 * Full `hipSOLVER` support
 * Full `rocSPARSE` support
 * New options:
-  * `--amap` to hipify as much as possible, ignoring '--default-preprocessor' behavior
+  * `--amap` to hipify as much as possible, ignoring `--default-preprocessor` behavior
+  * `--clang-resource-directory` to specify the clang resource path - the path to the parent folder for the `include` folder,
+    containing `__clang_cuda_runtime_wrapper.h` and other header files used during hipification process
```

Comment:
Recommended

contains `__clang_cuda_runtime_wrapper.h` and other header files used during hipification process